controllers/articles.controllers.js

```diff
@@ -9,4 +11,15 @@ const getArticleById = (request, response, next) => {
       next(err);
     });
 };
-module.exports = getArticleById;
+
+const getAllArticles = (request, response, next) => {
+  fetchAllArticles()
+    .then((articles) => {
+      response.status(200).send(articles);
```

Comment:
articles need returned on a suitably named key not just sent back as the response body itself `.send({articles});` would do the trick but you will need to rewrite the tests first.